extension/cart/addProductsToCart.js

```diff
@@ -53,9 +53,15 @@ module.exports = async (context, input) => {
     } catch (err) {
       if (err.errors && err.errors.line_items) {
         for (const [index, lineItem] of Object.entries(err.errors.line_items)) {
+          if (lineItem.hasOwnProperty('variant_id') !== true) {
+            console.log('CONTINUE')
```

Comment:
I am sure a nicer description would be better :D

Reply:
Yeah, removed it. ^^